src/components/shared/ButtonWithPanel/ArrowPanel.css

```diff
@@ -4,7 +4,7 @@
 
 .arrowPanelAnchor {
   position: absolute;
-  z-index: 10;
+  z-index: 5;
```

Comment:
May I ask why this change? I mean, I'm not opposed :-) but this doesn't seem useful.

Reply:
This change was just meant to put z-indexes in order. The element below this one has an index of 4, so I just ordered them that way, so that we don’t use arbitrary numbers.
However, I’m not sure if it was intentionally given a large number. If so, we can definitely change it back to 10 :)

Reply:
yeah I think this is good, more "ordered". Especially if we sue "10" this should be explained better than just an arbitrary number, so in this case "5" is just as good. Thanks :-)

Projects that are a bit more in order than our project for z-index do have a clear plan about them. They also keep some "space" so that it's possible to add elements between the stacks without changing everything. We're not there yet!